lib/src/components/Icon/icons.ts

```diff
@@ -14,6 +14,8 @@ export const arrows = [
 ] as const
 
 export const actions = [
```

Comment:
Should keep these arrays in alphabetic order – easier to scan, no decisions to make when adding new icon. 🤓